draft-ietf-scone-protocol.md

```diff
@@ -550,8 +659,51 @@ discards a SCONE packet without also successfully processing another packet
 from the same datagram SHOULD ignore any throughput advice signal. Such a datagram
 might be entirely spoofed.
 
+Once negotiated, endpoints that seek to receive throughput advice on a flow
+MUST send a SCONE packet at least twice each monitoring period; see {{time}}.
+
+Sending SCONE packets more often might be necessary to:
```

Comment:
I'm trying to understand the consequence of not conforming to the above MUST (2x/period).  I assume it's not "or the NE drops the flow".

I read this bit and the section that follow as why a sender should send than 2x/period, but maybe its just line 665 that is misleading me?